pkg/cli/topo.go

```diff
@@ -271,18 +249,18 @@ func getWatchAllCommand() *cobra.Command {
 }
 
 func runWatchEntityCommand(cmd *cobra.Command, args []string) error {
-	return watch(cmd, args, "entity")
+	return watch(cmd, args, topo.Object_ENTITY)
 }
 
 func runWatchRelationCommand(cmd *cobra.Command, args []string) error {
-	return watch(cmd, args, "relation")
+	return watch(cmd, args, topo.Object_RELATIONSHIP)
 }
 
 func runWatchAllCommand(cmd *cobra.Command, args []string) error {
-	return watch(cmd, args, "all")
+	return watch(cmd, args, topo.Object_UNSPECIFIED)
 }
 
-func watch(cmd *cobra.Command, args []string, filter string) error {
+func watch(cmd *cobra.Command, args []string, objectType topo.Object_Type) error {
 	replay, _ := cmd.Flags().GetBool("replay")
```

Comment:
I think the default for the CLI should be `replay` on - It might make sense to call the flag `noreplay`